pages/MainTabs.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { createBottomTabNavigator } from '@react-navigation/bottom-tabs';
+import { Text, View } from 'react-native';
+import HomeScreen from './HomeScreen';
+import BottomMenuBar from '../components/BottomMenuBar';
+import ShowScheduleScreen from './ShowScheduleScreen';
+
+const Tab = createBottomTabNavigator<any>();
+
+function PlaceholderScreen({ title }: { title: string }) {
+  return (
+    <View style={{ flex: 1, alignItems: 'center', justifyContent: 'center' }}>
+      <Text>{title}</Text>
+    </View>
+  );
+}
+
+export default function MainTabs() {
+  return (
+    <Tab.Navigator screenOptions={{ headerShown: false }} tabBar={(props) => <BottomMenuBar {...props} />}>
```

Comment:
```suggestion
    <Tab.Navigator screenOptions={someHeaderShownObj} tabBar={BottomMenuBar}>
```

_should_ work for most good impls of a component, if TS complains try an `as any` (or check into the type it wants for that prop and `as That`)